src/components/PageBlocks/GeometryBase.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <wrap
+    v-bind="$props"
+    v-on="$listeners"
+  >
+    <div
+      v-if="processing"
+      class="d-flex align-items-center justify-content-center h-100"
+    >
+      <b-spinner />
+    </div>
+    <template v-else>
+      <l-map
+        ref="map"
+        :zoom="map.zoom"
+        :center="map.center"
+        :min-zoom="map.zoomMin"
+        :max-zoom="map.zoomMax"
+        :bounds="map.bounds"
+        class="w-100"
+        style="height: 100%;"
+        :max-bounds="map.lockBounds ? map.bounds : null"
+      >
+        <l-tile-layer
+          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+          :attribution="map.attribution"
+        />
+        <l-polygon
+          v-for="(geometry, i) in geometries"
```

Comment:
Display markers from same feed in polygon if the feed is configured like that.

Make sure you don't mix polygons from different feeds